Gemfile.lock

```diff
@@ -262,8 +260,8 @@ GEM
     responders (3.0.1)
       actionpack (>= 5.0)
       railties (>= 5.0)
-    rexml (3.2.4)
-    rubocop (1.8.1)
+    rexml (3.2.5)
```

Comment:
### [rexml](https://github.com/ruby/rexml)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.2.4 → 3.2.5

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **56** commits. See the full changes on [the compare page](https://github.com/ruby/rexml/compare/v3.2.4...v3.2.5).

These are the first 10 commits:
- (0ddb7ee) [Bump version](https://github.com/ruby/rexml/commit/0ddb7ee01a4684d2ffa375f87b1326167e659eea)
- (be62163) [travis: update Ruby versions](https://github.com/ruby/rexml/commit/be62163ba12a6657679a34e472b1d29d75e0e881)
- (af10ea5) [Already `$SAFE` has been removed in master](https://github.com/ruby/rexml/commit/af10ea538e3be1622f7dfb69b2eca44b63c1f9f2)
- (ad5de4e) [xpath: add an error check for unprocessed rest data](https://github.com/ruby/rexml/commit/ad5de4e78e7c10803952fd898ed742c2fbf7e865)
- (119e388) [xpath: add more invalid cases](https://github.com/ruby/rexml/commit/119e38840c6ef6014f5f069d1102c7b2be74ad00)
- (c1db4d5) [`require "rexml/document"` by default (#36)](https://github.com/ruby/rexml/commit/c1db4d5ce9e06381d63e1fbf44f6f5a65bcc248e)
- (d97b318) [Use Refinements to define dclone methods that are used only …](https://github.com/ruby/rexml/commit/d97b318acb2658a48d94f666d49dd6218aa1f256)
- (8c5d5cd) [Exclude useless files after installation (#39)](https://github.com/ruby/rexml/commit/8c5d5cd75c9f15a77e1a2a0162a141f8ee85a8b1)
- (c306201) [Test on ruby 3.0 (#40)](https://github.com/ruby/rexml/commit/c306201e55b75eb0c8f8ccca33b154b3cb50140c)
- (1e59077) [ci: migrate to GitHub Actions from Travis CI](https://github.com/ruby/rexml/commit/1e59077896912a7a35be45f14ce95770f4d6b158)


</details>